assets/queries/openAPI/2.0/multi_collectionformat_not_valid_in_parameter/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	[path, value] := walk(doc)
+	param := value.parameters[n]
+	param.collectionFormat == "multi"
+	param.in != "query"
+	param.in != "formData"
```

Comment:
shouldn't it be `param.in != "query"` OR `param.in != "formData"` ? The way it is looks like an `AND` to me

Reply:
No, this query should not be positive if in is query or formData, so we want to dispatch an issue if in is not query AND not formData